Use alt-super-c for switch color filter instead of alt-super-s comonly used to toogle
the screen reader on or off in several distributions shipping the Mate desktop- Didier Spaier
diff -ru compiz-plugins-main-master/metadata/colorfilter.xml.in compiz-plugins-main-master.new/metadata/colorfilter.xml.in
--- compiz-plugins-main-master/metadata/colorfilter.xml.in	2021-10-30 17:36:42.000000000 +0200
+++ compiz-plugins-main-master.new/metadata/colorfilter.xml.in	2024-04-10 22:35:56.118244911 +0200
@@ -23,7 +23,7 @@
       <option name="switch_filter_key" type="key">
         <short>Switch filter</short>
         <long>Select filter to use from filters list : either apply all filters or only one specific filter (triggering this action switches filters one by one and eventually comes back to the "all filters" mode).</long>
-        <default>&lt;Super&gt;&lt;Alt&gt;s</default>
+        <default>&lt;Super&gt;&lt;Alt&gt;c</default>
       </option>
     </display>
     <screen>
